Project.toml

```diff
@@ -18,12 +18,12 @@ RigidBodyDynamics = "366cf18f-59d5-5db9-a4de-86a9f6786172"
 [compat]
 ColorTypes = "0.7, 0.8, 0.9, 0.10"
 CoordinateTransformations = "0.5, 0.6"
-GeometryTypes = "0.6, 0.7, 0.8"
+GeometryBasics = "0.2, 0.3"
 InteractBase = "0.8, 0.9, 0.10"
 Interpolations = "0.9, 0.10, 0.11, 0.12"
 LoopThrottle = "0.1"
-MechanismGeometries = "0.4, 0.5"
-MeshCat = "0.10, 0.11"
+MechanismGeometries = "0.4, 0.5, 0.6"
+MeshCat = "0.10, 0.11, 0.13"
```

Comment:
Should this be just `MeshCat = "0.13"`? Changing the dependency from GeometryTypes.jl to GeometryBasics.jl means this would not work for MeshCat 0.10 and 0.11, right?

Reply:
Yeah, that's right--this can just be `0.13` now